### PR TITLE
Housekeeping chores

### PR DESCRIPTION
**Pull request by leekelleher**
Whilst I was doing the other PRs, went a bit OCD on the VS project...

- Copied over the ".editorconfig" from Umbraco-CMS, to keep with the same coding style/formatting
- Copies over ".gitattributes" file (for consistency between repos)
- `CTRL+K,D` the code files, for whitespace formatting
- Removed unused assembly references, (less hard dependencies)
- Updated "AssemblyInfo.cs" details/copyright
- Included the root text files (README, etc) as solution items in the VS project, (for quicker access)
- Alphabetical order of the property-editor value-connectors in the README.

### CODE REVIEW

**Comment by clausjensen** [comment]
Hey @leekelleher 
Thanks for all of the PRs!
Just to give you an update - I'll be merging these next week 😄

File: README.md [hunk 1]
```diff
@@ -12,21 +12,22 @@ Primarily this project offers connectors for the most popular Umbraco community
...
 
+- [Archetype](https://our.umbraco.org/projects/backoffice-extensions/archetype/)
 - [DocType Grid Editor](https://our.umbraco.org/projects/backoffice-extensions/doc-type-grid-editor/)
-- [UrlPicker](https://our.umbraco.org/projects/backoffice-extensions/urlpicker/)
 - [LeBlender](https://our.umbraco.org/projects/backoffice-extensions/leblender)
 - [Nested Content](https://our.umbraco.org/projects/backoffice-extensions/nested-content/)
-- [Stacked Content](https://github.com/umco/umbraco-stacked-content)
 - [nuPickers](https://our.umbraco.org/projects/backoffice-extensions/nupickers/)
-- [Archetype](https://our.umbraco.org/projects/backoffice-extensions/archetype/)
+- [Multi Url Picker](https://our.umbraco.com/projects/backoffice-extensions/multi-url-picker)
```
Comment (clausjensen) [inline README.md]: The OCD part of me loves everything about this PR 😄 
... you however missed one (Multi Url Picker) when sorting! 😀

Comment (leekelleher) [reply README.md]: Bah! 😆 I'll update it.

Comment (clausjensen) [reply README.md]: Nah its okay I've fixed and merged 😄

**Comment by clausjensen** [comment]
@leekelleher all done! 😃 
Awesome work and thanks for contributing.
I've reviewed the changes to the existing connectors and it all looks good.

I haven't tested the new connectors you have added since that would require too much time to actually get it done right now + my guess is that you already tested these when working on them... I'd also assume you would rather have them in there now, than having to wait for me to test them 😄
As I see it - having _something_ in there is better than having nothing.

**Comment by leekelleher** [comment]
Thanks @clausjensen!

I've done initial testing, _only for pretty basic scenarios though - I didn't go crazy nested-gridified-vorto-content-lists._ 😆

As you say, something is better than nothing - we can patch up any unexpected issues. 👍

**Pull request merged**